standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -3585,6 +3532,77 @@ public ShowCompactResponse showCompact(ShowCompactRequest rqst) throws MetaExcep
     }
   }
 
+  @RetrySemantics.ReadOnly
+  public GetLatestCompactionResponse getLatestCompaction(GetLatestCompactionRequest rqst) throws MetaException {
+    GetLatestCompactionResponse response = new GetLatestCompactionResponse();
+    response.setDbname(rqst.getDbname());
+    response.setTablename(rqst.getTablename());
+    Connection dbConn = null;
+    PreparedStatement pst = null;
+    try {
```

Comment:
You can use try-with-resources, also don't forget to close the resultset

Reply:
In the latest change, I closed the connection, statement, resultset in the finally block as some other methods in this file.